### ISSUE TITLE
Adding dummy routes

### ISSUE THREAD

**Issue by plato79** [govizlora/optical-breacher#32]
Sometimes you need to add a dummy route so you could access proper order afterwards.. 

For example if the top part doesn't have the required starter, sometimes you need to select a dummy one than continue from there.. 

It is also possible that you need to add this dummy part in the middle. Could you check if it's possible to do this?

**Comment by govizlora** [comment]
What do you mean by "dummy"? My understanding is that if the required route is `A -> B -> C - D` and if you go `E -> A-> B -> C -> D` then `E` is the dummy one, correct?

**Comment by plato79** [comment]
That's what I meant, yes. Sometimes, you need to put dummy between segments also.. Like `A -> B -> E -> C -> D`...

When it says there are no solutions, I could find a solution with this route.

**Comment by govizlora** [comment]
When the required route is `A -> B -> C - D`, will `A -> B -> E -> C -> D` be valid?

**Comment by plato79** [comment]
Yes, if you have enough buffer, ABECD also solves "AB" & "CD", of course if the route is ABCD you need to follow without missing a step.

**Comment by govizlora** [comment]
Thanks. I think currently the app should be able to solve such problems with no issue. For example, if the required sequence is `ABCD`, the app can find answers like `EABCD`. Can you provide a photo or screenshot when it's not working as desired, so I can further investigate it?
